services/director-v2/src/simcore_service_director_v2/core/settings.py

```diff
@@ -100,13 +102,28 @@ def endpoint(self) -> str:
         return self.S3_ENDPOINT
 
 
+class StorageSettings(BaseCustomSettings):
```

Comment:
At line 390 you already have `STORAGE_ENDPOINT`. Can we avoid having it two times?

Reply:
removed STORAGE_ENDPOINT